Meetups/23/2025-02-20-minutes.md

```diff
@@ -0,0 +1,170 @@
+
+Andreas: thanks to standards and JSON schema.
+
+Henk: What discovery mechanism did you use? Very impressing presentation, and I can tell there were a lot of work.
+
+Andreas: To discover a new thing I will just input the Thing Description URL. In terms of the protocols we only support the easy stuff URL and JSON.
+
+Henk: Are you planning any "automatic loading" of TDs?
+
+Andreas: we have a registry CSV or a north bound API.
+
+Andreas: there are devices protected by password. We added a small extension to TD to express the role and the accounts who can use the device.
+
+Henk: If you have 5 people with different roles, do they access to different part of the system? or they can't do some actions?
+
+Andreas: on the database level there is a table which contains all the things and two other tables connected with Properties and Actions. We can lock different part of this table to different roles. there for we can support hiding or reviling Properties or Actions you can view and use.
+
+Andreas: we are interested working with the community and love to hear more from you. Please use our github repository. The project is at his beginning.
+
+Ege: any inputs for the standardization process?
+
+Andreas: we are still working of some of the feature. But from the research perspective is semantic harmonization. It would be wonderful to use JSON-LD and semantic more to automate this process.
+
+Vignesh: We there is something that we need to used but is not in the standard is it okay to be added in the TD?
+
+Andreas: we ignore them, but they don't show in the UI. You can make extension of the WoT manager to use this information.
+
+Cris: are you storing timeseries in the postgres database
+
+Andreas: we are storing the latest value, but it is easy to change.
+
+Cris: is it polling based?
+
+Andreas: yes.
+
+Ege: Interesting point. Describing which properties needs to be stored in a timeseries db could be something we can do in the specification.
+
+Andreas: whether to keep the value or not is definitely something not device specific.
+
+Andreas: what do think about the AI agent? For me the interesting part is that you don't have to think about different application context.
+
+Ege: it is interesting that you showed how you can achieve a good interaction with the AI with off the shelf tools without reinviting any AI specific. Probably, in this case you can use quite small AI models to achieve good results.
+
+Andreas: we pre process the TD before send it to the LLM.
+
+Cris: what are the limitations?
+
+Andreas: in complex contexts it gets messy.
+
+Cris: are you polishing the AI output?
+
+Andreas: currently is directly connected.
+
+Cris: do you have some Role access control for AI agents?
+
+Andreas: User privileges are in there but they use a different key.
+
+Ege: Using AI prompts used in same way can help solving the data augmentation problem?
+
+Andreas: not really, the data mapping should be very precise. But it could be a good idea.
```

Comment:
```suggestion
Andreas: Currently it is directly connected.

Cris: Do you have some Role-based access control for AI agents?

Andreas: User privileges are in there, but they use a different key.

Ege: Can using AI prompts help solve the data augmentation problem?

Andreas: Not really, the data mapping should be very precise. But it could be a good idea.
```